packages/nimble/inst/CppCode/predefinedNimbleLists.cpp

```diff
@@ -246,12 +247,14 @@ void OptimControlNimbleList::copyFromSEXP(SEXP S_nimList_) {
     SET_STRING_ELT(S_pxData, 0, mkChar(".xData"));
     PROTECT(S_trace = findVarInFrame(GET_SLOT(S_nimList_, S_pxData),
```

Comment:
Ah I see in `cppDefs_nimbleList.R`. This issue is independent of this PR, so I'll fix that in a separate PR.